uportal-war/src/main/java/org/jasig/portal/groups/pags/dao/jpa/PersonAttributesGroupTestDefinitionImpl.java

```diff
@@ -37,6 +37,7 @@
 import javax.persistence.TableGenerator;
 import javax.persistence.Version;
 
+import org.apache.commons.lang.StringUtils;
```

Comment:
Prefer `lang3` when introducing the new import.  Sooner or later uPortal will stop depending upon commons lang < v3.